core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/AutoTuner.scala

```diff
@@ -935,6 +936,7 @@ object AutoTuner extends Logging {
   val DEF_READ_SIZE_THRESHOLD = 100 * 1024L * 1024L * 1024L
   val DEFAULT_WORKER_INFO_PATH = "./worker_info.yaml"
   val SUPPORTED_SIZE_UNITS: Seq[String] = Seq("b", "k", "m", "g", "t", "p")
+  val DEFAULT_PLATFORM = "onprem"
```

Comment:
I am not sure if this right place to define the constant.
It probably should be defined as a constant of the profiler because the description of that argument does not suggest it's scope is the autoTuner. In the future, we will find some usage to that argument within the profiling logic.

Reply:
Moved it to Profiler.